exporters/prometheus/src/main/java/io/opentelemetry/exporter/prometheus/Otel2PrometheusConverter.java

```diff
@@ -400,29 +402,44 @@ private Exemplars convertDoubleExemplars(List<DoubleExemplarData> exemplars) {
     return Exemplars.of(result);
   }
 
+  @Nullable
   private Exemplar convertExemplar(double value, ExemplarData exemplar) {
     SpanContext spanContext = exemplar.getSpanContext();
+    Labels labels = Labels.EMPTY;
     if (spanContext.isValid()) {
-      return new Exemplar(
-          value,
+      labels =
           convertAttributes(
               null, // resource attributes are only copied for point's attributes
               null, // scope attributes are only needed for point's attributes
               exemplar.getFilteredAttributes(),
               "trace_id",
               spanContext.getTraceId(),
               "span_id",
-              spanContext.getSpanId()),
-          exemplar.getEpochNanos() / NANOS_PER_MILLISECOND);
+              spanContext.getSpanId());
     } else {
-      return new Exemplar(
-          value,
-          convertAttributes(
-              null, // resource attributes are only copied for point's attributes
-              null, // scope attributes are only needed for point's attributes
-              exemplar.getFilteredAttributes()),
-          exemplar.getEpochNanos() / NANOS_PER_MILLISECOND);
+      labels = convertAttributes(null, null, exemplar.getFilteredAttributes());
+    }
+    int codePoints = getCodePoints(labels);
+    if (codePoints > EXEMPLAR_MAX_CODE_POINTS) {
+      THROTTLING_LOGGER.log(
+          Level.WARNING,
+          "exemplar labels have "
+              + codePoints
+              + " codePoints, exceeding the limit of "
```

Comment:
i'd replace `codePoints` with `unicode code points` to make it easier to understand what this warning means

Reply:
@laurit  addressed.

> Task :exporters:prometheus:test

```
Otel2PrometheusConverterTest > exemplarLabelsAboveLimit() STANDARD_ERROR
    [Test worker] WARN io.opentelemetry.exporter.prometheus.Otel2PrometheusConverter - exemplar labels have 193 unicode code points, exceeding the limit of 128
```